sparql-ns.ttl

```diff
@@ -197,17 +197,17 @@ sparql:lang rdf:type sparql:Function ;
     .
 
 sparql:langdir rdf:type sparql:Function ;
-    rdfs:comment "" ;
+    rdfs:comment "This function returns the base direction of a literal. " ;
     rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-langdir> ;
     .
 
 sparql:haslang rdf:type sparql:Function ;
-    rdfs:comment "This function returns true if the given RDF literal has a specified language, matching the literal’s language tag. It returns false otherwise." ;
+    rdfs:comment "This function returns true if the given RDF literal has a specified language, matching the literal’s language tag." ;
     rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-haslang> ;
     .
 
 sparql:haslangdir rdf:type sparql:Function ;
-    rdfs:comment "" ;
+    rdfs:comment "THis fun ction rutrns true if the given RDF literal has an initial base direction." ;
```

Comment:
It's unfortunate PR #204 was merged so quickly after [this commit](https://github.com/w3c/sparql-query/pull/204/commits/4d257ffbae68631f2d99ae77c8d06b468943e65e).

```suggestion
    rdfs:comment "This function returns true if the given RDF literal has an initial base direction." ;
```